changelog.d/223.change.rst

```diff
@@ -0,0 +1,4 @@
+``attrs``-decoratorated classes are never modified now.
+Instead a new class is created with the appropriated methods and attributes attached from the get go.
+
+For backward-compatibility, the behavior of getting hashing by ID on Python 3 when passing ``@attr.s(hash=False)`` is manually retained by setting ``__hash__`` to ``object.__hash__``.
```

Comment:
Something is grammatically off here… maybe:

For backward-compatibility, the prior behavior of hashing by ID on Python 3 when passing ``@attr.s(hash=False)`` may be retained retained by setting ``__hash__`` to ``object.__hash__``.

Reply:
Oh dear, so my wording was so bad that it confused you what I mean. 🙈

*We* do set `__hash__` to `object.__hash__` automatically, to not break compat.

English is hard man.  Let me try again…

> The previous (incorrect) behavior of hashing by object ID on Python 3 when passing `@attr.s(hash=False)` (which implies `cmp=True`), is retained  for backward compatibility reasons.
>
> A better -- and more explicit -- way to control object hashing is in the pipeline.

Better/clearer?

Reply:
Ah, yes, OK.  :-)